### PR TITLE
feat(windows): add langtags.json data

### PR DESCRIPTION
**Pull request by mcdurdin**
This will be used for future updates for language search
and installation and will eventually allow us to remove
ethnologue and alltags data (and some of bcp47 subtag)

### CODE REVIEW

File: resources/standards-data/langtags/readme.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# langtags.json
+
+This file is retrieved from https://ldml.api.sil.org/langtags.json
```
Comment (darcywong00) [inline resources/standards-data/langtags/readme.md:3]: Is this file manually retrieved?

File: resources/standards-data/langtags/readme.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+This file lists implicit script tags for language tags and equivalent BCP 47 tags per SLDR (and hence CLDR). We
+use this to build up good clean BCP 47 tags for adding languages to Windows.
+
+This file is intended to replace alltags, ethnologue, and iso639-3 data files in future. language-subtag-registry includes extra script metadata not present in langtags.json.
+
+## File Format
+
+* Source repository: https://github.com/silnrsi/langtags/
+* Documentation: https://github.com/silnrsi/langtags/blob/master/doc/langtags.md
+* Unofficial schema: https://gist.github.com/mcdurdin/eff65bb2d77034525120d48cbe280d30
```
Comment (darcywong00) [inline resources/standards-data/langtags/readme.md:14]: The attached langtags.json has API 1.2.1

Should the unofficial schema include the following?
iana [Always an array 1.2.0]
obsolete: boolean [Added 1.2.1]
unwritten: boolean [Added 1.2.1]

(These would be ignored in BuildLangTags.pas)

File: windows/src/unit-tests/standards-data/Keyman.Test.System.Standards.LangTagsRegistryTest.pas [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+  Assert.AreEqual('PG', TLangTagsMap.LangTags['akq'].region);
+  Assert.AreEqual(False, TLangTagsMap.LangTags['akq'].suppress);
+  Assert.AreEqual('akq-Latn', TLangTagsMap.LangTags['akq'].windows);
+
+  // All Tags
+
+  Assert.AreEqual('arx', TLangTagsMap.AllTags['arx-BR']);
+  Assert.AreEqual('bfi-x-stokoe', TLangTagsMap.AllTags['sgn-bfi-GB-x-stokoe']);
+  Assert.AreEqual('bfy', TLangTagsMap.AllTags['ppa-Deva-IN']);
+end;
```
Comment (darcywong00) [inline windows/src/unit-tests/standards-data/Keyman.Test.System.Standards.LangTagsRegistryTest.pas:58]: If we continue to use `und` (und-fonipa?) for sil_ipa, can we add tests for
```
    {
        "full": "und-Zyyy-001",
        "iana": [ "Undetermined" ],
        "iso639_3": "und",
        "name": "Undetermined",
        "region": "001",
        "regionname": "World",
        "script": "Zyyy",
        "sldr": false,
        "tag": "und",
        "tags": [ "und-001", "und-Zyyy" ],
        "windows": "und-Zyyy"
    },
```

Comment (darcywong00) [reply windows/src/unit-tests/standards-data/Keyman.Test.System.Standards.LangTagsRegistryTest.pas:58]: and are there any special cases to check for Amharic or Tigrinya?

File: resources/standards-data/langtags/readme.md (hunk 1 above)
Comment (mcdurdin) [reply resources/standards-data/langtags/readme.md:3]: Yes, that's intentional at this point.

File: resources/standards-data/langtags/readme.md (hunk 2 above)
Comment (mcdurdin) [reply resources/standards-data/langtags/readme.md:14]: Yes, the unofficial schema does need updating, but wanting to hand that over to Martin and have it added to the langtags repo.

File: windows/src/unit-tests/standards-data/Keyman.Test.System.Standards.LangTagsRegistryTest.pas (hunk 3 above)
Comment (mcdurdin) [reply windows/src/unit-tests/standards-data/Keyman.Test.System.Standards.LangTagsRegistryTest.pas:68]: > and are there any special cases to check for Amharic or Tigrinya?

I don't think so, no.

**Review by darcywong00 — APPROVED** [review]
lgtm

**Pull request merged**